### PR TITLE
Refresh onboarding documents

### PR DESCRIPTION
**Pull request by rmtolmach**
I was going through my onboarding checklist here: https://github.com/department-of-veterans-affairs/va.gov-team/issues/35142 and found a couple broken links. I fixed or removed them and did a little bit of cleaning up.

I have a couple open questions. I will make comments in the PR to make it clear.

### CODE REVIEW

File: teams/vsp/teams/operations/daytime-oncall-support-role.md [hunk 1]
```diff
@@ -3,40 +3,42 @@
...
 
 * Normal oncall [responsibilities](https://github.com/department-of-veterans-affairs/va.gov-team-sensitive/tree/master/OnCall) 
-* Monitoring oncall channel
+* Monitoring #oncall channel (and others listed below)
 * Requests for assistance from the VFS or VSP teams
-* Support requests in the support channels
+* Support requests in the support channel (#vfs-platform-support)
 * Responding to and reviewing any internal or external DevOps PRs
-* Respond and grant [approved access](https://github.com/department-of-veterans-affairs/devops/blob/master/docs/operations/Internal_Onboarding_Instructions.md) to requests for access in Zenhub or remove access per [offboarding requests](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/working-with-vsp/contractoroffboarding.md).
+* Respond and grant approved access (**is there an updated link to add here?**) to requests for access in Zenhub or remove access per [offboarding requests](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/working-with-vsp/contractoroffboarding.md)
```
Comment (rmtolmach) [inline teams/vsp/teams/operations/daytime-oncall-support-role.md:10]: The original link 404s 
[approved access](https://github.com/department-of-veterans-affairs/devops/blob/master/docs/operations/Internal_Onboarding_Instructions.md)
Is there something else we want to put here?

File: teams/vsp/teams/operations/daytime-oncall-support-role.md [hunk 2]
```diff
@@ -54,14 +56,16 @@ Your Github team notifications and [#vsp-operations](https://dsva.slack.com/chan
...
 
 ### Working on Postmortem action items
-In Zenhub there is a label entitled "Postmortem Action Item". These are tasks that have been identified after an incident's postmortem. The oncall Dev should work on these items If there is no active support work or incident. If there is another dev working on the Postmortem Action Items connect with them to try and see how to help get that work completed. 
+In Zenhub there is a label entitled "Postmortem Action Item". These are tasks that have been identified after an incident's postmortem. The oncall Dev should work on these items If there is no active support work or incident. If there is another dev working on the Postmortem Action Items, connect with them to try and see how to help get that work completed.
 
 ### Supporting and Monitoring Deployments
 If deploys fail and there is no direct owner, take some time to try and find the cause of the failed deploy. If you do decide to look into a failed deployment make sure to start a thread under it and let others know. 
 
+Can we list some Slack channels or links to Jenkins here that Oncall should monitor?
```
Comment (rmtolmach) [inline teams/vsp/teams/operations/daytime-oncall-support-role.md:67]: Question here. 

Maybe the main oncall doc would be a better place to call them out?
https://github.com/department-of-veterans-affairs/va.gov-team-sensitive/tree/master/OnCall

File: teams/vsp/teams/operations/onboarding.md [hunk 3]
```diff
@@ -1,18 +1,18 @@
...
 
 Speaking of VAEC and how things fit together, some introductions to the organization [here](https://github.com/department-of-veterans-affairs/devops/blob/master/docs/VA-IT-Organization-and-Contacts.md).
 
-There are currently 3 environments for the applications we deploy: dev, staging, and prod. A fourth is coming called sandbox. We also deploy into a utility environment for tools that don't fit cleanly that we only have one of (Jenkins, Grafana, etc).
+There are currently four environments for the applications we deploy: dev, staging, prod and sandbox. We also deploy into a utility environment for tools that don't fit cleanly that we only have one of (Jenkins, Grafana, etc).
 
-On the application side there's a (somewhat older, but still relevant) video of [how endpoints get added to the API](https://www.youtube.com/watch?v=V_i8JLXk5rg&feature=youtu.be) and some details about how the app works at that level.
+This video doesn't exist on youtube anymore :/ Is there something else we can link here?
```
Comment (rmtolmach) [inline teams/vsp/teams/operations/onboarding.md:15]: Question here.

File: teams/vsp/teams/operations/daytime-oncall-support-role.md (hunk 1 above)
Comment (jbritt1) [reply teams/vsp/teams/operations/daytime-oncall-support-role.md:10]: Perhaps https://vfs.atlassian.net/wiki/spaces/OT/pages/1792114735/Onboard+New+Team+Members+Granting+Access+to+Internal+Tools

Comment (rmtolmach) [reply teams/vsp/teams/operations/daytime-oncall-support-role.md:10]: Offboarding link is out of date but the  Offboarding Request template is valid

File: teams/vsp/teams/operations/onboarding.md (hunk 3 above)
Comment (jbritt1) [reply teams/vsp/teams/operations/onboarding.md:15]: Not sure on this one

Comment (jbritt1) [reply teams/vsp/teams/operations/onboarding.md:15]: Perhaps https://depo-platform-documentation.scrollhelp.site/developer-docs/Adding-a-new-External-Service-Integration.1884586162.html coupled with https://vfs.atlassian.net/wiki/spaces/OT/pages/1474594279/Forward+Proxy

File: teams/vsp/teams/operations/daytime-oncall-support-role.md (hunk 2 above)
Comment (jbritt1) [reply teams/vsp/teams/operations/daytime-oncall-support-role.md:67]: Those channels are captured in https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/vsp/teams/operations/daytime-oncall-support-role.md

**Comment by rmtolmach** [comment]
I pushed some updates. I included a commit that updates the template. Pinging @jhouse-solvd for his review of that document.

File: .github/ISSUE_TEMPLATE/ops_onboarding_checklist.md [hunk 4]
```diff
@@ -58,17 +58,12 @@ High level:
...
-  - [ ] Faith
-  - [ ] Sri
-  - [ ] Ryan
-  - [ ] Robert
-  - [ ] Gary
-  - [ ] Troy
-  - [ ] Azeez
+  - [ ] John
+  - [ ] Oseas
+
```
Comment (jsgarmon) [inline .github/ISSUE_TEMPLATE/ops_onboarding_checklist.md:65]: Wow, we have slimmed down quite a bit. Make sure to include yourself!

Comment (rmtolmach) [reply .github/ISSUE_TEMPLATE/ops_onboarding_checklist.md:66]: oops! done ✅

File: teams/vsp/teams/operations/daytime-oncall-support-role.md [hunk 5]
```diff
@@ -54,13 +56,13 @@ Your Github team notifications and [#vsp-operations](https://dsva.slack.com/chan
...
+Create an issue describing the request in GitHub and link it to one of the two Super Epics. Assign it to yourself. Track any work done with comments. Only track support that takes more than 15 mins and if it is a PR that takes under 15 mins please link to the super epic. If the support work seems to be taking you down a long road, make a note around this and bring to the team to scope out into a project. Unless an emergency, don't spend more than 2 hours on any given support work. Summarize both failed attempts and final solutions so the issue can be referred to again in the future if necessary. If other team members are involved, add them to the ticket.
```
Comment (jhouse-solvd) [inline teams/vsp/teams/operations/daytime-oncall-support-role.md:59]: let's change this to 30 minutes. 

and "Unless **it's** an emergency"

**Review by jhouse-solvd — CHANGES_REQUESTED** [review]
your attention to detail is fantastic. great updates. I added one small comment. if you can edit and resubmit, then we're good to go.

File: teams/vsp/teams/operations/daytime-oncall-support-role.md [hunk 6]
```diff
@@ -54,13 +56,13 @@ Your Github team notifications and [#vsp-operations](https://dsva.slack.com/chan
...
+If a support request takes more than 30 mins of your time, create a GitHub Issue describing the request and link it to one of the two Super Epics. Assign it to yourself. Track any work done with comments. If a support request takes less than 30 minutes and results in a PR, please link the PR to the super epic. If the support work seems to be taking you down a long road, make a note around this and bring to the team to scope out into a project. Unless it's an emergency, don't spend more than 2 hours on any given support work. Summarize both failed attempts and final solutions so the issue can be referred to again in the future if necessary. If other team members are involved, add them to the ticket.
```
Comment (rmtolmach) [inline teams/vsp/teams/operations/daytime-oncall-support-role.md:59]: @jhouse-solvd I made some more changes to this paragraph to make it clearer. But... I'm not sure if the changes are accurate. Can you re-check this paragraph?

Comment (jhouse-solvd) [reply teams/vsp/teams/operations/daytime-oncall-support-role.md:59]: This is good. I'm creating an issue to document support request guidance. That guidance can then be linked into this template. But, that's outside the scope of your pr. :)

**Comment by jhouse-solvd** [comment]
Looks good to me!

**Review by jhouse-solvd — APPROVED** [review]

**Pull request merged**